src/kinova_gen3_mujoco_config/description/mujoco/gen3_7dof_mujoco.xml

```diff
@@ -282,13 +285,13 @@
     hence scale = 0.8 * 100 / 255
   -->
   <actuator>
-    <general class="2f85" name="fingers_actuator" tendon="split" forcerange="-5 5" ctrlrange="0 255"
-      gainprm="0.3137255 0 0" biasprm="0 -100 -10"/>
+    <general class="2f85" name="robotiq_85_left_knuckle_joint" tendon="split" forcerange="-5 5" ctrlrange="0 0.8"
+      gainprm="100" biasprm="0 -100 -10"/>
   </actuator>
```

Comment:
hmm, the comment above makes it seem like `ctrlrange` should be `[0, 255]`, but the link that's referenced in the comment is outdated. I tried to look around the docs to see if I could find anything about this, but no luck. Maybe having `ctrlrange` be from `[0, 255]` is an outdated requirement? I'm not sure. Do you or @pac48 have any thoughts about this?